spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/filter/factory/rewrite/CachedBodyOutputMessage.java

```diff
@@ -32,7 +32,7 @@
 /**
  * Implementation of {@link ClientHttpRequest} that saves body as a field.
  */
-class CachedBodyOutputMessage implements ReactiveHttpOutputMessage {
+public class CachedBodyOutputMessage implements ReactiveHttpOutputMessage {
```

Comment:
@spencergibb @TYsewyn 
I've added the public modifier but will remove if any objections present